pkg/globalmanager/controllers/incrementallearning/incrementallearningjob.go

```diff
@@ -815,9 +819,83 @@ func (c *Controller) createInferPod(job *sednav1.IncrementalLearningJob) error {
 	return err
 }
 
+func (c *Controller) appendStatusCondition(name, namespace string, cond sednav1.ILJobCondition) error {
```

Comment:
it's better to extract function `appendStatusCondition` as a common funtion?

Reply:
Agreed, but it's not trival since different CRs have different object clients which is hard to merge as one client, and it is not problem of this pr. So leave it alone.